src/common/pages/onboard.tsx

```diff
@@ -680,40 +705,47 @@ const Onboard = (props: Props) => {
                 </>
               )}
 
-            <div className="onboard-delegate-rc">
-              <div className="col-span-12 sm:col-span-10">
-                <div className="onboard-check mb-2">
-                  <input 
-                    type="checkbox" 
-                    className="onboard-checkbox" 
-                    checked={isChecked} 
-                    onChange={() => {
-                      setChecked(!isChecked)
-                    }}
-                  />
-                  <span className="onboard-blinking-text">Delegate some resource credits to {decodedInfo && decodedInfo!.username} (Minimum Rc is 5Bn)</span>
-                </div>
-                {isChecked &&
-                <div className="mt-3">
-                  {(rcAmount && rcError) ? <span className="text-danger mt-3">{rcError}</span> : ""}
-                  <InputGroup>
-                    <FormControl
-                      type="text"
-                      placeholder={"Enter amount to delegate(Bn)"}
-                      value={rcAmount}
-                      onChange={(e: any) => setRcAmount(Number(e.target.value))}
+              <div className="onboard-delegate-rc">
+                <div className="col-span-12 sm:col-span-10">
+                  <div className="onboard-check mb-2">
+                    <input
+                      type="checkbox"
+                      className="onboard-checkbox"
+                      checked={isChecked}
+                      onChange={() => {
+                        setChecked(!isChecked);
+                      }}
                     />
-                  </InputGroup>
-                  <div className="operation-amount d-flex mt-3">
-                    <span className="operations">Posts/Comment: {commentAmount} |</span>
-                    <span className="operations">Votes: {voteAmount} |</span>
-                    <span className="operations">Transfers: {transferAmount} |</span>
-                    <span className="operations">Reblogs/ Follows: {customJsonAmount}</span>
+                    <span className="onboard-blinking-text">
+                      Delegate some resource credits to {decodedInfo && decodedInfo!.username}{" "}
```

Comment:
@Adesojisouljay these should be added to translation strings

Reply:
> @Adesojisouljay these should be added to translation strings

Alright, that's noted.